webkit-release

```diff
@@ -778,6 +835,9 @@ if __name__ == '__main__':
         prog = 'webkit-release',
         usage = '%(prog)s [options ...] command [command-options ...] tarball')
 
+    parser.add_argument('-p', '--port', dest = 'port', default = "gtk", choices = ("gtk", "wpe"),
```

Comment:
I think it should be easy to autodetect the port form the tarball instead of having to pass --port to the script

Reply:
Sure, I can look into doing that. Should this be detected based on the tarball name, or from its contents? I think the tarball names will always match `^wpewebkit-.*` and `^webkitgtk-.*`.

Reply:
Whatever is easier for you :-)